app/web/features/dashboard/Hero/HeroButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { ThemeProvider } from "@material-ui/core";
+import Button from "components/Button";
+import { DASHBOARD } from "i18n/namespaces";
+import Link from "next/link";
+import { useTranslation } from "next-i18next";
+import { searchRoute } from "routes";
+import makeStyles from "utils/makeStyles";
+
+import useHeroBackgroundTheme from "./useHeroBackgroundTheme";
+
+const useStyles = makeStyles((theme) => ({
+  textGradient: {
+    background: `-webkit-linear-gradient(0deg, ${theme.palette.primary.main}, ${theme.palette.secondary.main})`,
+    "-webkit-background-clip": "text",
+    "-webkit-text-fill-color": "transparent",
```

Comment:
Interesting that this has pretty good cross-browser support, and yet only available with the `webkit` prefix 🤔